tests/plugins/emoji/dropdown.js

```diff
@@ -28,6 +28,22 @@
 		setUp: function() {
 			bender.tools.ignoreUnsupportedEnvironment( 'emoji' );
 		},
+
+		// This test should be on top of test suite, cause other tests will cache emojis (#2583).
+		'test emoji names cache': function() {
+			var bot = this.editorBot,
+				collision = CKEDITOR.tools.array.filter( bot.editor._.emoji.list, function( item ) {
+				return item.id === ':collision:';
+			} )[ 0 ];
+
+			assert.isUndefined( collision._name, 'Emoji name should be undefined.' );
```

Comment:
I think you missed this `_name` entry.

```suggestion
			assert.isUndefined( collision.name, 'Emoji name should be undefined.' );
```